frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SlideInOptions.java

```diff
@@ -2,17 +2,15 @@
 
 import lombok.Getter;
 import lombok.experimental.Accessors;
-import org.checkerframework.checker.tainting.qual.Untainted;
 
 @Getter
 public class SlideInOptions {
 
-  @Untainted
   private String name;
   @Accessors(fluent = true)
   private Boolean shouldReattach;
 
-  public SlideInOptions(@Untainted String name, boolean shouldReattach) {
+  public SlideInOptions(String name, boolean shouldReattach) {
```

Comment:
I think you can safely assume the result of `com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeyUnmanagedBranchName#getNameOfUnmanagedBranch` which gets passed here to be `@Untainted` aka non-HTML

Reply:
Or rather, not _safely assume_ but _explicitly ensure_, as per https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1152#discussion_r986184061

Reply:
as a branch name may include character which could be understood as html tags I think at this point we cannot say it's html-free. possible html escaping should be done closer to the usage